### PR TITLE
SOMMELIER-12

### PR DESCRIPTION
**Pull request by constantkek**

### CODE REVIEW

**Comment by Mixxx-a** [comment]
### YOU MERGE IT TO MASTER!

File: package.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
     "eject": "rnstl && expo eject",
     "storybook": "start-storybook -p 7007",
     "build-storybook": "build-storybook",
-    "prettier": "prettier --write 'App/**/*.{ts,tsx,js}'",
+    "prettier": "prettier --write App/**/*.{ts,tsx,js}",
     "prestorybook": "rnstl",
-    "lint": "eslint './App/**/*.{js,ts,tsx}'"
+    "lint": "eslint ./App/**/*.{js,ts,tsx}"
```
Comment (Mixxx-a) [inline package.json]: Are you sure that this will also work on MAC?

File: App/components/Image/image.styled.ts [hunk 2]
```diff
@@ -1,16 +1,14 @@
 import styled from 'styled-components/native'
+import { BorderType } from '../Box/box.types'
 
 type ImageContainerProps = {
-  image: string
   width?: number
   height?: number
-  border?: string
   borderRadius?: number
 }
```
Comment (Mixxx-a) [inline App/components/Image/image.styled.ts]: Shouldn't be there all props (where is image source prop?)

**Review by Mixxx-a — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (constantkek) [reply package.json]: @Mixxx-a idk but that's a way for windows. Probably should check it

File: App/components/Image/image.styled.ts (hunk 2 above)
Comment (constantkek) [reply App/components/Image/image.styled.ts]: in image.tsx. Set as source={{ uri: image }} so we dont need it here in styled file

Comment (constantkek) [reply App/components/Image/image.styled.ts]: border is also set in image.tsx but the image is wrapped into box

File: App/components/Image/Image.story.tsx [hunk 3]
```diff
@@ -16,7 +14,32 @@ const defaultImage = () => (
     )}
     width={number('width', 128)}
     height={number('height', 128)}
-    border={select('border', BORDERS, `5px ${BORDERS[2]} gray`)}
+    border={{
```
Comment (StanislavMyakishev) [inline App/components/Image/Image.story.tsx]: Border props have changed in the major box update, consider removing props here it according to the recent changes 😄

**Review by Mixxx-a — CHANGES_REQUESTED** [review]
Is this still works after recent `Box` changes? For example, now we don't have `BorderType` in `'../Box/box.types'` but you are importing it in `image.styled.ts`

**Comment by constantkek** [comment]
@Mixxx-a yeah that's shitty, I'll fix this

**Review by Mixxx-a — COMMENTED** [review]
I will test you component)

**Review by Mixxx-a — APPROVED** [review]

**Review by StanislavMyakishev — APPROVED** [review]

File: App/components/Image/Image.story.tsx (hunk 3 above)
Comment (constantkek) [reply App/components/Image/Image.story.tsx]: @StanislavMyakishev whoops, this part wasnt added to commit smh. Fixed

Comment (constantkek) [reply App/components/Image/Image.story.tsx]: yeah just forget to save file in vscode 😄

**Pull request merged**